lib/board.rb

```diff
@@ -82,9 +82,17 @@ def correct_placement_length?(ship, placement_coordinates)
     ship.length == placement_coordinates.length
   end
 
+  def all_cells_available?(placement_coordinates) 
+    # iterate through placement coordinates and check that the cells are empty?
+
+    placement_coordinates.all? do |coordinate|
+      cells[coordinate].empty?
+    end
+  end
+
   # consider renaming
```

Comment:
I really feel like we should rename this now 😂

Reply:
Oh yes. Absolutely agree